tests/services/test_service.py

```diff
@@ -71,4 +74,32 @@ def test_audit_log_create_system_identity(app, db, service, resource_data):
         assert result["action"] == "draft.create"
         assert result["resource"]["id"] == "abcd-1234"
         assert result["resource"]["type"] == "record"
-        assert result["user"]["id"] == system_user_id
+        assert result["user"]["id"] == "1"
+
+
+def test_audit_log_builder(app, client, db, service, current_user):
+    """Should succeed when creating an audit log via AuditLogBuilder using unit of work."""
+    with app.test_request_context():
+        current_user.login(client)
```

Comment:
TODO: Fix login in test (the tests will pass then)

Reply:
check for `client_with_login` fixtures and `login_user` in the tests of other modules. Especially the ones which test resource layer.

Reply:
Thank you! The tests are fixed now.